package.json

```diff
@@ -44,15 +44,15 @@
     "commit": "ui-scripts --commit",
     "install:react": "ui-scripts --install-react",
     "install:packages": "lerna bootstrap --reject-cycles",
-    "bootstrap": "npm-run-all clean:modules install:packages clean build build:tokens build:types",
+    "bootstrap": "node scripts/bootstrap.js",
     "bootstrap:docs": "npm-run-all clean:modules install:packages clean build",
     "build": "lerna run build --stream",
     "build:watch": "lerna run build:watch --stream",
     "build:docs": "lerna run bundle --stream --scope docs-app",
     "build:types": "lerna run build:types --stream",
     "build:tokens": "ui-token-scripts generate-all-tokens",
-    "build:ts": "yarn build && yarn build:types",
-    "build:node-scripts": "lerna run build --scope @instructure/ui-scripts --include-dependencies",
+    "build:ts": "lerna run prepare-build --stream --scope @instructure/ui-icons && tsc -b tsconfig.references.json --verbose",
```

Comment:
This is just a question: is it possible to somehow filter or color code the output of tsc? We have so many packages, and really hard to pinpoint from the log what packages were updated and what packages weren't. It's a nice-to-have, but otherwise seems to work fine :)

Reply:
There should be a way to somehow color the output of tsc, but TBH I'm not sure how at the moment, so if it is okay for you then I will create a ticket for this and investigate a little bit on how to do it.